packages/web-components/src/custom-elements.ts

```diff
@@ -36,6 +36,7 @@ import { fluentTextField } from './text-field/index';
 import { fluentTooltip } from './tooltip/index';
 import { fluentTreeView } from './tree-view/index';
 import { fluentTreeItem } from './tree-item/index';
+import { Container } from '@microsoft/fast-foundation';
```

Comment:
@EisenbergEffect looks like the lint portion of the PR build is throwing on the import order here. This likely needs to move to the top (line 3)

Reply:
On it.